hiclass/_calibration/VennAbersCalibrator.py

```diff
@@ -0,0 +1,320 @@
+import numpy as np
+from sklearn.model_selection import StratifiedKFold
+from hiclass._calibration.BinaryCalibrator import _BinaryCalibrator
+from scipy.stats import gmean
+from hiclass._calibration.calibration_utils import _one_vs_rest_split
+from collections import defaultdict
+from sklearn.utils.validation import check_is_fitted
+from sklearn.base import BaseEstimator
+from hiclass._hiclass_utils import _normalize_probabilities
+
+
+class _InductiveVennAbersCalibrator(_BinaryCalibrator):
+    name = "InductiveVennAbersCalibrator"
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def fit(self, y: np.ndarray, scores: np.ndarray, X: np.ndarray = None):
+        positive_label = 1
+        unique_labels = np.unique(y)
+        assert len(unique_labels) <= 2
+
+        y = np.where(y == positive_label, 1, 0)
+        y = y.reshape(-1)  # make sure it's a 1D array
+
```

Comment:
Maybe these lines can be replaced with the `binary_only` estimator tag https://scikit-learn.org/stable/developers/develop.html#estimator-tags

```suggestion
```

Reply:
I can't use estimator tags on this class because it does not extend BaseEstimator